### PR TITLE
Alt/swell mainnet

### PR DESCRIPTION
**Pull request by Wazabie**
# Adding a New Chain
This PR adds Swell Mainnet (namely Swellchain on the Ethereum List) to the registry.

## Checklist
- [x] I have declared the chain at the appropriate [Superchain Level](../docs/glossary.md#superchain-level-and-rollup-stage).
- [x] I have run `just validate <chain-id>` locally to ensure all local validation checks pass. 
- [x] I have run `just codegen` to ensure that the chainlist and other generated files are up-to-date and include my chain.

### CODE REVIEW

**Comment by tessr** [comment]
@Wazabie Can you confirm that this PR matches the one opened by AltLayer (https://github.com/ethereum-optimism/superchain-registry/pull/713) but with a chain name that matches the ethereum-lists/chains source of truth?

**Comment by Wazabie** [comment]
@tessr that is correct, I'll close #713 in favour of this one

**Review by sbvegan — APPROVED** [review]
lgtm

File: superchain/configs/mainnet/swell.toml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+sequencer_rpc = "https://swell-mainnet.alt.technology"
+explorer = "https://explorer.swellnetwork.io"
+superchain_level = 0
+standard_chain_candidate = true # This is a temporary field which causes most of the standard validation checks to run on this chain
+batch_inbox_addr = "0x005dE5857e38dFD703a1725c0900E9C6f24cbdE0"
+canyon_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+delta_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+ecotone_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+fjord_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
+granite_time = 0 # Thu 1 Jan 1970 00:00:00 UTC
```
Comment (zhwrd) [inline superchain/configs/mainnet/swell.toml:13]: is there a reason why we arent using `superchain_time = 0` so it follows superchain hardfork times?

**Pull request merged**